src/packageData.js

```diff
@@ -245,9 +245,17 @@ const packages = Object.assign(
 
 const latestPackages = new Set(Object.values(packages));
 
+const packagesWarnAboutEnv = new Set([
+  'babel-preset-es2015',
+  'babel-preset-es2016',
+  'babel-preset-es2017',
+  'babel-preset-latest'
+]);
+
 module.exports = {
   packages,
   presets,
   plugins,
   latestPackages,
+  packagesWarnAboutEnv
```

Comment:
Where is this used?

Reply:
hm not sure what this is, I suspect I pushed it up by accident

Reply:
ah yeah i had started looking at the `Log when replacing out preset-es2015,16,17,latest as FYI` mentioned in the readme at some point and then accidentally added these changes to my commit